### ISSUE TITLE
Investigate why playwright reports are not included in build output

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#7605]
The playwright a11y report is what is used to show a11y statuses from our website, however, these reports are not included in our build output. This requires us to manually generate the report and include that file in our website, a manual step that we haven't been doing causing the a11y statuses on our website to become outdated.

We should identify any missing steps to ensure this report is included in our build output so that the a11y status from our website shows the most up to date information.
